doc/articles/faq.md

```diff
@@ -33,7 +33,9 @@ You can develop Uno Platform applications on Windows, macOS, or Linux. Supported
 
 Much like the new UI technologies from Microsoft, Uno Platform doesn’t support creation of new applications using VB.NET.
 
-If you have an existing VB.NET application which you would like to port/modernize for cross-platform scenarios with Uno Platform, you should be able to reuse all of your VB.NET business logic, packaged as NET standard libraries into a new Uno Platform app where only the new UI code would be defined in XAML with some glue in C#. Additionally, If you’d like to move any of your VB.NET code to C# you may be able to use automated tools such as https://converter.telerik.com
+If you have an existing VB.NET application which you would like to port/modernize for cross-platform scenarios with Uno Platform, you should be able to reuse all of your VB.NET business logic, packaged as NET standard libraries into a new Uno Platform app where only the new UI code would be defined in XAML with some glue in C#.
+To be exact, add "Class Library" VB project (not "Class Library (.Net Framework)", and not "Class Library (Universal Windows)"). When you want to use this library for UWP app compatible with phones, you have to change Target Framework to ".Net Standard 1.4" (in Project:Properties), as this is last .Net Standard that can be used with projects targetting Windows 10 15063. if you are targetting Windows 10 16299+, use .Net Standard 2.0. Then, in Uno heads for all platforms, add reference to this library (simplest way is to right click on "References" nodes inside these heads). You can use same Class Library also in your original, VB project. By the way, you can use this Class Library also in MAUI project :)
```

Comment:
```suggestion
To be exact, add "Class Library" VB project (not "Class Library (.Net Framework)", and not "Class Library (Universal Windows)"). When you want to use this library for UWP app compatible with phones, you have to change Target Framework to ".Net Standard 1.4" (in Project:Properties), as this is last .Net Standard that can be used with projects targetting Windows 10 15063. If you are targetting Windows 10 16299+, use .Net Standard 2.0. Then, in Uno heads for all platforms, add reference to this library (simplest way is to right click on "References" nodes inside these heads). You can use same Class Library also in your original, VB project. The same Class Library can also be used in MAUI project.
```

Reply:
I don't think it is necessary to mention MAUI as it just makes it more confusing (usually it is a FAQ itself - does Uno run on top of MAUI?). Furthermore - Windows Mobile is no longer supported, so I think that does not need to be mentioned either. Uno apps are compatible with phones by default, just not with Windows ones 🤷‍♂️

Reply:
Maybe I'm the last person using Windows Phone (i.e., Lumia 532 with Windows 10 15063) on the whole world, but also, maybe I'm not the last one. I created 38 apps, and most of them I'm still using on phone, and still updating.
And I don't want to have active two instances of these apps, one for my current phone, and second - for my possible new phone. I consider Uno as a platform to rescue these apps in case I have to switch to some newer, not Windows phone.
As this FAQ entry is about VB, mentioning MAUI can be important - as there is possibility that MAUI would support VB. Then porting app from VB\UWP to newer platform can be done in two ways: converting code to C#, not touching XAML, skipping Background working, Bluetooth, Contacts, SMS, etc.; or converting XAML, and not touching code unrelated to UI at all (seems like Bluetooth and similar items would be present in MAUI sooner than some PRs would be merged into Uno).

Reply:
Martin is right, we dont want to add confusion by mentioning external possibly compatible libraries at this point, given the history of running our WASM renderers on top of XF. The rest looks good.

Reply:
This does not seem resolved yet

Reply:
But this is now changed...

"To be exact, add "Class Library" VB project (not "Class Library (.Net Framework)", and not "Class Library (Universal Windows)"). When you want to use this library for UWP app compatible with phones (i.e., app that works on Windows Phones as Lumia), you have to change Target Framework to ".Net Standard 1.4" (in Project:Properties), as this is last .Net Standard that can be used with projects targetting Windows 10 15063. If you are targetting Windows 10 16299+, use .Net Standard 2.0. Then, in Uno heads for all platforms, add reference to this library (simplest way is to right click on "References" nodes inside these heads). You can use same Class Library also in your original, VB project. The same Class Library can also be used MAUI project, if you ever decide to port app from Uno to MAUI."
 
I would re-create this PR (in a new branch), to get rid of folder name casing.

Reply:
I see that the mention is still there, see comment from Francois above